javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,28 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+	const body = document.querySelector('body');
+	const ul = document.createElement('ul');
+	body.appendChild(ul);
+
+	for (let i =0;i < products.length; i++) {
+		const li = document.createElement('li');
```

Comment:
`const listItem` would be descriptive.

Reply:
ok thanks i will use the descriptive way